anago

```diff
@@ -1138,7 +1138,7 @@ Additional binary downloads are linked in the [%s](%s).'
 ##############################################################################
 # When creating new alphas on the master branch, make sure that X.Y-1.0
 # has been created first or warn.
-# This is to ensure that branchff can continue to be used as needed.  User
+# This is to ensure that `krel ff` can continue to be used as needed.  User
 # can override here.
 # @param buildver - Incoming JENKINS_BUILD_VERSION
 branchff_sanity_check () {
```

Comment:
`s/branchff_sanity_check/fastforward_check`

Reply:
Renamed it :+1: